expression/aggregation/base_func_test.go

```diff
@@ -1,41 +1,44 @@
 package aggregation
 
 import (
-	"github.com/pingcap/check"
+	"testing"
+
 	"github.com/pingcap/parser/ast"
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/expression"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/mock"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = check.Suite(&testBaseFuncSuite{})
-
-type testBaseFuncSuite struct {
+type mockBaseFuncSuite struct {
 	ctx sessionctx.Context
 }
 
-func (s *testBaseFuncSuite) SetUpSuite(c *check.C) {
+func createBaseFuncSuite(t *testing.T) (s *mockBaseFuncSuite) {
+	s = new(mockBaseFuncSuite)
 	s.ctx = mock.NewContext()
+	return
 }
```

Comment:
This suite is nit, you can just inline the logic.

Reply:
Fixed it - ed1ada7

Reply:
Fixed it - 2aea1c8